src/FondOfSpryker/Yves/GoogleTagManager/ControllerEventHandler/Cart/RemoveProductControllerEventHandler.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+
+namespace FondOfSpryker\Yves\GoogleTagManager\ControllerEventHandler\Cart;
+
+use FondOfSpryker\Shared\GoogleTagManager\EnhancedEcommerceConstants;
+use FondOfSpryker\Yves\GoogleTagManager\ControllerEventHandler\ControllerEventHandlerInterface;
+use FondOfSpryker\Yves\GoogleTagManager\Dependency\Client\GoogleTagManagerToCartClientInterface;
+use FondOfSpryker\Yves\GoogleTagManager\Session\EnhancedEcommerceSessionHandlerInterface;
+use Generated\Shared\Transfer\EnhancedEcommerceProductDataTransfer;
+use Generated\Shared\Transfer\ItemTransfer;
+use Symfony\Component\HttpFoundation\Request;
+
+class RemoveProductControllerEventHandler implements ControllerEventHandlerInterface
+{
+    /**
+     * @var \FondOfSpryker\Yves\GoogleTagManager\Session\EnhancedEcommerceSessionHandlerInterface
+     */
+    protected $sessionHandler;
+
+    /**
+     * @var \FondOfSpryker\Yves\GoogleTagManager\Dependency\Client\GoogleTagManagerToCartClientInterface
+     */
+    protected $cartClient;
+
+    /**
+     * @param \FondOfSpryker\Yves\GoogleTagManager\Session\EnhancedEcommerceSessionHandlerInterface $sessionHandler
+     * @param \FondOfSpryker\Yves\GoogleTagManager\Dependency\Client\GoogleTagManagerToCartClientInterface $cartClient
+     */
+    public function __construct(
+        EnhancedEcommerceSessionHandlerInterface $sessionHandler,
+        GoogleTagManagerToCartClientInterface $cartClient
+    ) {
+        $this->sessionHandler = $sessionHandler;
+        $this->cartClient = $cartClient;
+    }
+
+    /**
+     * @return string
+     */
+    public function getMethodName(): string
+    {
+        return 'removeAction';
+    }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param string $locale
+     *
+     * @return void
+     */
+    public function handle(Request $request, string $locale): void
+    {
+        $sku = $request->get(EnhancedEcommerceConstants::PRODUCT_FIELD_SKU);
+
+        if (!$sku) {
```

Comment:
prüfst du hier auf null oder liefert get auch bool zurück?

Reply:
Wie oben, in dem Feld kann theoretisch alles mögliche drin sein. Für mich ist an der Stelle nur wichtig das der Parameter gesetzt ist. Im folgenden Verlauf prüfe ich dann auf valide SKU